src/ros/ardiunotopython/test.py

```diff
@@ -0,0 +1,25 @@
+import serial
```

Comment:
Add a short description to this file. Also make sure to include copy write statement as needed.

Reply:
I'm not sure if there's any for it. It's kinda original one. So many ardiuno programmers use very similar this one. It's from pyserial. I'm going to add the description

Reply:
Updated